feature-group-definitions/text-wrap-balance.yml

```
@@ -1,4 +1,5 @@
 name: "text-wrap: balance"
+description: "The `text-wrap: balance` CSS declaration wraps text content across lines, while using an algorithm that balances that number of characters on each line, enhancing legibility."
```

```suggestion
description: "The `text-wrap: balance` CSS property value balances the length of each line when text is broken into multiple lines. Also known as headline balancing."
```

(I don't think "CSS property value" is great, but we have it already, and I'd like to change them all together in a single PR.)

---

Usually, when I talk about these things, I say that `text-wrap` is the property, that `balance` is the value, and that `text-wrap: balance` is the declaration. I don't really like the phrase "property value" because I always someone would be confused by it and think we're only talking about `balance`, or only about `text-wrap`.

That being said, I agree that it's better to be consistent for now. As you said, we can always bulk-change if we think we have a better proposal in the future.